src/main/java/Ui.java

```diff
@@ -0,0 +1,44 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+public class Ui {
+    private Scanner sc;
+    private static final String INTRO = "Hi! I'm ELON MUSK\nWhat can I do for you?";
```

Comment:
Great that you've abstracted print lines into the Ui class! To prevent long sentences, perhaps you can shorten it via string concatenation into a new line, so something like their duke logo. 
![Capture](https://user-images.githubusercontent.com/88648894/187116468-6fb6a756-3039-4922-a70e-098d7909aa09.JPG)

I'm not sure that using System.out.println in multiple classes is the best idea? Maybe consider using a Stringbuilder instead of printing to console, so you only do one final `system.out.println`. Might help for this week's task, where you have to consolidate ElonMusk's response into one string